Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"\nCorona Spread Prediction\n\n@author David Hammbartsumjan\n@author Tilman Schlichting\n@since 27.10.2021\n@copyright CC BY-SA\n\n\"\"\"\nfrom ruleset import ruleset\nfrom simulation import sim\n\nclass Main:\n    def __init__(self):\n        \"\"\"Konstruktor\"\"\"\n        self.config = open('config.json')\n\n\n# Intitialize runtime\nruntime = Main()\n\n# Create a ruleset with Paramenters from JSON\nruntime.ruleset = ruleset.Ruleset(runtime.config).getRuleset()\n\n#Create a Simulation with given Rulesets\nruntime.sim = sim.Sim(runtime.ruleset)\n\n#Debugpoint\nprint(\"hold\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 00c0aef2e9e1bad078649fb2994c0fcebdfa0311)
+++ b/main.py	(date 1635351986257)
@@ -25,5 +25,7 @@
 #Create a Simulation with given Rulesets
 runtime.sim = sim.Sim(runtime.ruleset)
 
+#diagramme
+
 #Debugpoint
 print("hold")
\ No newline at end of file
